utils/defs.tex

```diff
@@ -167,7 +167,7 @@
 \definecolor{gray}{rgb}{0.92,0.92,0.92}
 
 \lstset{ % set defaults for languages not otherwise defined
-  breakatwhitespace=false,         % sets if automatic breaks should only happen at whitespace
+  breakatwhitespace=true,         % sets if automatic breaks should only happen at whitespace
   basicstyle=\ttfamily\footnotesize,
   breaklines=true,                 % sets automatic line breaking
   escapeinside={|}{|},          % if you want to add LaTeX within your code
```

Comment:
@jdinan I wonder whether this is the source of the LaTeX rendering issues for the logical OR operator `||`.

Reply:
Yes, this seems to be the issue. According to my build, the `escapeinside={|}{|},` line can be removed safely or the delimiters can be changed.

Reply:
Yes, this is the source of the problem.  I thought I had fixed it by changing the escapeinside for only the ```apiexamples``` ```lstset```, but I apparently didn't commit the change.  Unless another example is using this (I don't think we currently have escapes in any examples, but I also didn't check), it would be safest to just remove it entirely.

Reply:
I ran `grep -F '|' example_code/*.*` and only turned up your one example. :+1: for removing the line entirely.

Reply:
Listings documentation indicates that escapeinside defaults to nothing (no escapes), so it should be safe to not specify an escape sequence.  :+1: